XivAlexander/App_ConfigRepository.h

```diff
@@ -111,12 +111,13 @@ namespace App {
 		ConfigItem<bool> AlwaysOnTop{ this, "AlwaysOnTop", false };
 		ConfigItem<bool> UseHighLatencyMitigation{ this, "UseHighLatencyMitigation", true };
 		ConfigItem<bool> UseHighLatencyMitigationLogging{ this, "UseHighLatencyMitigationLogging", true };
+		ConfigItem<bool> UseLatencyCorrection{ this, "UseLatencyCorrection", true };
+		ConfigItem<int> BaseLatencyPenalty{ this, "BaseLatencyPenalty", 30, [](int newValue) {
```

Comment:
Isn't this value always lower the better? Even then, lowering this value to 1 still won't let you do stuff which also is impossible on <1ms ping, so I think it's better to just hardcode it to 1.

Reply:
The lower the better, yes. However I think having it at 30 as a baseline is more for adding a safety buffer in the event the program reduces too much.

Maybe the setting should just be 1 by default and leave that judgement to the user? Calculating penalty by ping deviation is quite safe on its own too.